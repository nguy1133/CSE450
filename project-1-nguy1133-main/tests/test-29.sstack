// A longer case for CONCAT and SPLIT (partially from project example)
PUSH "This is a "
PUSH "sentence"
CONCAT
POP full
PUSH full
PRINT full
SPLIT " "
SPLIT " "
SPLIT " "
CONCAT " new "
CONCAT
PRINT  // Should print "a new sentence"
CONCAT " "
CONCAT
PRINT  // Should print "This is"
