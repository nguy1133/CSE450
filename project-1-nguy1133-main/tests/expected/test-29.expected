This is a sentence
a new sentence
This is
